systemdspawner/systemd.py

```diff
@@ -143,6 +144,20 @@ async def start_transient_service(
         )
         run_cmd.append(f"--property=EnvironmentFile={environment_file}")
 
+    # make sure cmd[0] is absolute, which is a requirement of systemd-run
```

Comment:
This shouldn't be necessary [since systemd v207](https://github.com/systemd/systemd/blob/477fdc5afed0457c43d01f3d7ace7209f81d3995/NEWS#L13327) for systemd-run and in service files [since v239](https://github.com/systemd/systemd/blob/477fdc5afed0457c43d01f3d7ace7209f81d3995/NEWS#L7173).

Reply:
It's not required for it to be absolute _at all_, but it is required to be absolute to be correct, because systemd-run ignores $PATH, instead using its own internal search path. Fixed the comment to be more precise.